controller/test-files/work_item/update/area/area_wit.res.payload.golden.json

```diff
@@ -13,15 +13,15 @@
             "kind": "area"
           }
         },
-        "system.area": {
+        "system_area": {
```

Comment:
The UI potentially makes use of these field information. @sudsen, do you parse the WIT description that the backend returns and do you search for fields inside of it?

Reply:
See https://github.com/fabric8-services/fabric8-wit/pull/2333#discussion_r230679076

Reply:
@jarifibrahim In the comment you referred to @sudsen said that the UI ignores `_` fields:

> UI will simply ignore '_' values for now.

But here you're **replacing** a field and the UI will not be able to find the `area`, right?

Reply:
Correct. This PR replaces `system.area` with `system_area` because `/workitemtype` response isn't fixed in this PR. I have changed in separately in https://github.com/fabric8-services/fabric8-wit/pull/2335 (see the golden file https://github.com/fabric8-services/fabric8-wit/pull/2335/files#diff-d743767476fe0f409baa3070f7728acc)